### PR TITLE
Disabled support tools if cluster is not registered.

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

- Redirect to register page when not register vars are set
- Added double column for register page

## How does it look?

Screen Shot 2022-07-12 at 14 41 53
Screen Shot 2022-07-12 at 14 41 46
Screen Shot 2022-07-12 at 14 41 38
Screen Shot 2022-07-12 at 14 36 50
Screen Shot 2022-07-12 at 14 36 43
Screen Shot 2022-07-12 at 14 36 35
Screen Shot 2022-07-12 at 14 36 28

Signed-off-by: Benjamin Perez <benjamin@bexsoft.net>

### CODE REVIEW

**Comment by bexsoft** [comment]
Updated screens:
Screen Shot 2022-07-13 at 17 42 44
Screen Shot 2022-07-13 at 17 42 27

File: portal-ui/src/screens/Console/Support/Register.tsx [hunk 1]
```diff
@@ -344,440 +344,457 @@ const Register = ({ classes }: IRegister) => {
...
-                https://subnet.min.io/cluster/register
-              </Link>
+                SUBNET
+              </Link>{" "}
+              to avail support for this MinIO cluster
             </Box>
-          </Box>
+          </Grid>
+        ) : null}
+        <Box
```
[the remark this answers is not in the document]
Comment (bexsoft) [reply portal-ui/src/screens/Console/Support/Register.tsx:632]: Yes, The idea is that user can change subscription parameters in case it is needed

**Review by reivaj05 — APPROVED** [review]

**Comment by harshavardhana** [comment]
Does this mean that an unregistered cluster can't run speedtest?

**Comment by harshavardhana** [comment]
> Does this mean that an unregistered cluster can't run speedtest?

any answer to my question?

**Comment by bexsoft** [comment]
> Does this mean that an unregistered cluster can't run speedtest?

Yes, that's correct

**Comment by harshavardhana** [comment]
> Yes, that's correct

Isn't this going to be a breaking change and perhaps a large change for even POC customers? - are we saying that even POCs will have to be registered?

**Comment by bexsoft** [comment]
> > Yes, that's correct
> 
> Isn't this going to be a breaking change and perhaps a large change for even POC customers? - are we saying that even POCs will have to be registered?

I see, yeah the requirement was to request for register the cluster to enable the tools actions. I think we might need to review this further.

**Comment by harshavardhana** [comment]
Is this clarified today? @bexsoft

**Comment by bexsoft** [comment]
> Is this clarified today? @bexsoft

Yes Harsha, We will be implementing a couple of changes on this & mc as well. PR will continue blocked until then

**Review by cniackz — COMMENTED** [review]
This PR is blocked, no need for a review right now.

**Review by reivaj05 — APPROVED** [review]

**Comment by bexsoft** [comment]
> This PR is blocked, no need for a review right now.

PR Ready for review

**Review by Alevsk — APPROVED** [review]

**Pull request merged**